pkg/manager/member/tiflash_member_manager.go

```diff
@@ -720,6 +727,11 @@ func (m *tiflashMemberManager) setStoreLabelsForTiFlash(tc *v1alpha1.TidbCluster
 	// for unit test
 	setCount := 0
 
+	if !tc.TiFlashBootStrapped() {
```

Comment:
What about using TiKVBootStrapped here?
Because `GetStores` will return normally only if TiKV is bootstrapped.

Reply:
OK, that's reasonable